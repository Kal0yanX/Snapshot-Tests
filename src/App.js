import './App.css';
import GitHubCard from './components/GitHubCard';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h2>Testing using Screenshot</h2>
        <GitHubCard />
      </header>
    </div>
  );
}

export default App;
