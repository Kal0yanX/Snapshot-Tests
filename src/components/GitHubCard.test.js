import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';
import Card from 'react-bootstrap/Card';
import ListGroup from 'react-bootstrap/ListGroup';

test('renders a snapshot', () => {
  const tree = renderer.create(<App/>).toJSON()
  expect(tree).toMatchSnapshot()
});