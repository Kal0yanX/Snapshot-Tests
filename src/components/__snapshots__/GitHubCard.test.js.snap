// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    alt="@Kal0yanX"
    className="card-img-top"
    src="Profile.png"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      <h1>
        GitHub Profile
      </h1>
    </div>
    <p
      className="card-text"
    >
      Click below for a link to my GitHub Profile.
    </p>
  </div>
  <div
    className="list-group-flush list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      Snapshot Testing Page for Activity RR-8
    </div>
  </div>
  <div
    className="card-body"
  >
    <a
      className="card-link"
      href="https://github.com/Kal0yanX"
    >
      GitHub Link
    </a>
  </div>
</div>
`;
